site/en/tutorials/structured_data/imbalanced_data.ipynb

```diff
@@ -414,8 +441,22 @@
         "### Understanding useful metrics\n",
         "\n",
         "Notice that there are a few metrics defined above that can be computed by the model that will be helpful when evaluating the performance.\n",
+        "These can be divided into three groups.\n",
+        "\n",
+        "#### Metrics for probability predictions\n",
+        "\n",
+        "As we train our network with the cross entropy as a loss function, it is fully capable of predicting class probabilities, i.e. it is a probabilistic classifier.\n",
+        "Metrics that assess probabilistic predictions and that are, in fact, **proper scoring rules** are:\n",
```

Comment:
> **proper scoring rules**

This is the first I've seen this term, if it's worth mentioning we should give a brief description of what that means, and why it's important.

Reply:
I'll try to add a single sentence. Under "Read more" is a canonical reference. On top of that, I can recommend to read https://arxiv.org/abs/0912.0902 (knowing that scoring rules and scoring functions coincide for binary classification).